server/src/models/Review.js

```diff
@@ -0,0 +1,44 @@
+const Model = require("./Model.js")
+
+class Review extends Model {
+    static get tableName() {
+        return "reviews"
+    }
+    static get jsonSchema() {
+        return {
+            type: "object",
+            required: ["reviewText", "rating", "dogParkId", "userId"],
+            properties: {
+                reviewText: {type: "string", minLength: 20},
+                rating: {type: ["integer", "string"]},
+                dogParkId: {type: ["integer", "string"]},
+                userId: {type: ["integer", "string"]}
```

Comment:
add spaces inside curly braces
https://github.com/airbnb/javascript#whitespace--in-braces

```js
{ type: ["integer", "string"] }
```